ARMTemplateInstructions.md

```diff
@@ -0,0 +1,95 @@
+
+- To create the Enterprise app registration for data access from Split
+  
+	```azurecli-interactive
+	        az ad app create --display-name <appRegistrationName>
+	```
+- Output will show objectid in the oauth2Permissions. Copy the value for the object id and run the following command with the copied object id.
+  
+	```azurecli-interactive
+		az ad sp create --id <objectid>
+	```
+- Check for the created application registration in **Entra Id -> App Registrations** on Azure portal. Go to **Manifest** blade, update the 'isEnabled' on line 35 to false, save the manifest. Copy the following values from the manifest,  `id` at line2, `appId` at line 7 and `name` at line 26, keep these values for use in next steps.
+
+	![Manifest](./Images/ManifestMenu.png)
+
+- Replace the entire manifest with the content copied from manifest.json in the repo. Edit the manifest with the values copied in previous step. Following edits are to be made :
+    1. "id" on line 2
+    1. "appId" on line 7
+    1. Add the "appId" on line 43 resulting in `api://$appId`.
+    1. "name" of the app registration on line 55.
+        Save the manifest. You should see a manifest updated message.
+
+- Copy the following snippet on line 81, in the `preAuthorizedApplications`, and click save. You should get a manifest updated message.
+	```json
+	{
+		"appId": "d3e90440-4ec9-4e8b-878b-c89e889e9fbc",
+		"permissionIds": [
+			"b77a34fd-bafc-45e0-8907-a6098d55c56d"
+		]
+	},
+	{
+		"appId": "1e2401ea-428f-4575-9bbf-b301f7e1eb67",
+		"permissionIds": [
+			"b77a34fd-bafc-45e0-8907-a6098d55c56d"
+		]
+	},
+	{
+		"appId": "73b67c52-525b-4470-9c5c-1e02c60b8a05",
+		"permissionIds": [
+			"b77a34fd-bafc-45e0-8907-a6098d55c56d"
+		]
+	}
+	```
+- Click on **Overview** blade and copy the value for `Application (client) ID` in Essentials, save the value for later use. Open the linked Enterprise application in the `Managed application in local directory` in Essentials.
+
+	![Overview](./Images/Overview_EApp.png)
+
+-  In the Enterprise Application, you can now assign the users and/or groups to the app-roles in the application.
+
+   	![Users and groups](./Images/UsersinEapp.png)
+
+## ARM template to setup the Experimentation resources
+
+- Create a resource group for Experimentation, in the subscription enabled for Preview. Set the subscription as default subscription for the CLI.
+  
+	```azurecli-interactive
+		az group create --location <location> --name <resource_group_name>
+	```
+
+- Update the ExperimentationPreviewParams.json file for the ARM template in ExperimentationPreviewTemplate.json. In ExperimentationPreviewParams.json, edit the value for "SEWEntraApplicationId" with the `application id` saved in the App registration step. Update the "RGname" parameter to have the name of the resource group you created in previous step. Run the deployment command on cli.

```

Comment:
Tell the user they should first save ExperimentationPreviewParams.json and ExperimentationPreviewTemplate.json on their machine.